hadoop-ozone/dist/src/main/smoketest/commonlib.robot

```diff
@@ -31,12 +31,21 @@ Get test user principal
     ${instance} =       Execute                    hostname | sed 's/scm[0-9].org/scm/;s/scm[0-9]/scm/;s/om[0-9]/om/'
     [return]            ${user}/${instance}@EXAMPLE.COM
 
+Get Security Enabled From Config
+    Return From Keyword If    '${SECURITY_ENABLED}' != ''
+    ${value} =    Execute    ozone getconf confKey ozone.security.enabled
+    IF    '${value}' != 'true' and '${value}' != 'false'
+           ${value} =    Set Variable    false
+    END
+    Set Global Variable      ${SECURITY_ENABLED}     ${value}
+
 Kinit HTTP user
     Pass Execution If   '${SECURITY_ENABLED}' == 'false'    Skip in unsecure cluster
     ${principal} =      Get test user principal    HTTP
     Wait Until Keyword Succeeds      2min       10sec      Execute            kinit -k -t /etc/security/keytabs/HTTP.keytab ${principal}
 
 Kinit test user
+    Run Keyword         Get Security Enabled From Config
```

Comment:
`Run Keyword` is useful if the name of the keyword is dynamic.  Otherwise it's unnecessary.

```suggestion
    Get Security Enabled From Config
```